package.json

```diff
@@ -26,6 +26,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "json": "npx json-server --watch data/db.json --port 8000",
```

Comment:
do you need the double dash `--` here when using npx to run stuff?

Reply:
I haven't messed with the string of code, this is just what I found online.